cmd/scatr/version.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/spf13/cobra"
+)
+
+const version = "0.2.1"
+
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Returns the current SCATR version",
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("SCATR analyzer testing framework")
+		fmt.Println("  Version:", version)
+		fmt.Println()
+		fmt.Println("Environment:")
+		fmt.Println("  GOOS:  ", runtime.GOOS)
+		fmt.Println("  GOARCH:", runtime.GOARCH)
```

Comment:
For general users, isn't OS and ARCH better than GOOS and GOARCH?

Reply:
Yep. I'll do that.
```suggestion
		fmt.Println("  OS:  ", runtime.GOOS)
		fmt.Println("  ARCH:", runtime.GOARCH)
```